src/com/dabomstew/pkrandom/newgui/Bundle.properties

```diff
@@ -574,4 +574,8 @@ GUI.settingsStringTooNew=The settings string was generated with a newer randomiz
 GUI.settingsStringTooOld=The settings string was generated with an older randomizer version. It can only be used with version %s.
 GUI.settingsStringLoaded=Settings loaded from string.
 GUI.settingsStringOlder=This settings string was created by an older randomizer version.\nThe randomizer will attempt to open it anyway, but you should look out for new options added since you made it.
-GUI.pleaseUseTheLauncher=<html>The randomizer was not started via the launcher. You will not be able to load 3DS games into the randomizer.<br /><br />If you are seeing this message despite using the launcher, make sure that you are using the launcher bundled<br /> with version 4.0.2 (or later) of the randomizer.
\ No newline at end of file
+GUI.pleaseUseTheLauncher=<html>The randomizer was not started via the launcher. You will not be able to load 3DS games into the randomizer.<br /><br />If you are seeing this message despite using the launcher, make sure that you are using the launcher bundled<br /> with version 4.0.2 (or later) of the randomizer.
+GUI.tmFollowEvolutionsCheckBox.text=Follow Evolutions
+GUI.tmFollowEvolutionsCheckBox.toolTipText=Causes TM/HM compatibility to follow along with evolutions.
```

Comment:
This is minor but it would be nice to see these mirror something like `GUI.paFollowEvolutionsCheckBox.toolTipText`. So it would look something like "<html>When this is selected and TM compatibility is randomized...."
You could even insert line breaks with and explain how you gain more TMs/Tutors when evolving, and how the odds are this are affected by whether or not you prefer same type.

Reply:
How does this look?
![image](https://user-images.githubusercontent.com/15931642/109280024-5270ec80-77cf-11eb-9e8c-721209651456.png)
![image](https://user-images.githubusercontent.com/15931642/109280037-56047380-77cf-11eb-948b-65da068fc174.png)

Reply:
Looks good to me but I'm not the best with strings. I'll leave this unresolved for now so Ajarmar to take a look.

Reply:
Yeah, looks good